Resources/views/Role/list_roles.html.twig

```diff
@@ -30,10 +30,10 @@
                     <tbody>
                     {% for role in roles %}
                         {# @var role \eZ\Publish\API\Repository\Values\User\Role #}
-                        <tr>
-                            <td><a href="{{ path("admin_roleView", {"roleId": role.id}) }}">{{ role.identifier }}</a></td>
-                            <td>{{ role.id }}</td>
-                            <td>
+                        <tr class="ez-role">
+                            <td class="ez-role-name"><a href="{{ path("admin_roleView", {"roleId": role.id}) }}">{{ role.identifier }}</a></td>
+                            <td class="ez-role-id">{{ role.id }}</td>
+                            <td class="ez-role-edit-button">
```

Comment:
well, a class called `button` on something which is absolutely not a button, that's not good. You can add such class on the `a` but by following the naming convention of PlatformUI, ie `ez-button-role-edit`